frontend/src/app/main/ui/comments.cljs

```diff
@@ -1124,13 +1144,17 @@
 
         test-id     (str/join "-" (map :seqn (sort-by :seqn thread-group)))
 
+        wl          (mf/deref refs/workspace-local)
```

Comment:
if the `wl` value is only needed on time of executing the click event, the value can be derefed with plain deref on the click handler instead of creating this reactive deref that adds a watcher on workspace-local ref.